src/clusterfuzz/_internal/tests/core/bot/tasks/trials_test.py

```diff
@@ -40,6 +42,22 @@ def setUp(self):
 
     test_helpers.patch(self, ['random.random'])
 
+  def source_side_test(self, config_file_content, probability, app_name,
+                       app_args, trial_app_args):
+    """source side trials test template"""
```

Comment:
nit: Capitalize comments and end with dot.

Reply:
done